src/Turbo/templates/components/Frame.html.twig

```diff
@@ -0,0 +1,7 @@
+{% props id -%}
+
+<turbo-frame id="{{ id }}" {{- attributes }}>
+    {%- if block('content') is not empty -%}
+        {% block content %}{% endblock %}
+    {%- endif -%}
```

Comment:
```suggestion
    {%- block content %}{% endblock -%}
```

Or you will render the block twice (once to test if its result is empty, once to display the result)